projects/plugins/boost/app/assets/src/js/pages/cache-debug-log/cache-debug-log.tsx

```diff
@@ -0,0 +1,47 @@
+import { __ } from '@wordpress/i18n';
+import Header from '$layout/header/header';
+import { useDataSync } from '@automattic/jetpack-react-data-sync-client';
+import { z } from 'zod';
+import Footer from '$layout/footer/footer';
+import styles from './cache-debug-log.module.scss';
+import classNames from 'classnames';
+import { Button } from '@automattic/jetpack-components';
+
+const CacheDebugLog = () => {
+	const [ { data: debugLog } ] = useDataSync( 'jetpack_boost_ds', 'cache_debug_log', z.string(), {
+		query: {
+			// Keep refreshing the logs every 10 seconds
+			refetchInterval: 10000,
+		},
+	} );
```

Comment:
Can we move this to a separate file?

Reply:
I don't know if that would give any advantage. Can you explain your idea?

Reply:
It's a pattern we use elsewhere in the codebase. I'm not overly fussed either way.

Reply:
Actually @pyronaur raised that I made the wrong call here, and should've listened to @dilirity more.

Not only does it belong in a separate file, it belongs outside of `/pages/`. This is core logic for how a debug log is read, so it belongs in its respective feature in a store file.

To answer this question:

> I don't know if that would give any advantage. Can you explain your idea?

The advantage is that:
- In future, if another component needs to query the debug log, it won't get duplicate code for reading it
- It means that a page (a component in `/pages/`) doesn't need to know how to read the debug log, it uses the feature lib for it.
- It's consistent with how we're laying out other features.

Fix here: https://github.com/Automattic/jetpack/pull/35889